components/brave_wallet_ui/page/screens/onboarding/import-or-restore-wallet/import-or-restore-wallet.tsx

```diff
@@ -75,15 +74,6 @@ export const OnboardingImportOrRestoreWallet = () => {
             </CardButton>
           )}
 
-          {importableWallets?.isLegacyCryptoWalletsInitialized && (
-            <CardButton to={WalletRoutes.OnboardingImportLegacy}>
-              <CardButtonTextContainer>
-                <p>{getLocale('braveWalletImportFromLegacy')}</p>
-              </CardButtonTextContainer>
-              <LegacyWalletIcon />
-            </CardButton>
-          )}
-
           <LinkRow>
```

Comment:
Please remove this alias braveWalletImportFromLegacy -> IDS_BRAVE_WALLET_IMPORT_FROM_LEGACY

Reply:
Fixed in 47d7e5d066c708d001c7e49516eaa8e10f060d0c